test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -557,13 +555,25 @@ public void ScanPackageDirectoryWithCheckingCertificatesEnabledWillLoadPackageWi
 
             // Verify that package resolved successfully
             var libs = CurrentDynamoModel.LibraryServices.ImportedLibraries.ToList();
-            Assert.IsTrue(libs.Contains(Path.Combine(PackagesDirectorySigned, "Signed Package", "bin", "Package.dll")));
+            Assert.IsTrue(libs.Contains(Path.Combine(PackagesDirectorySigned, "Signed Package", "bin", "SignedPackage.dll")));
 
             // Verify that the package are imported successfully
             var entries = CurrentDynamoModel.SearchModel.SearchEntries.ToList();
-            Assert.IsTrue(entries.Any(x => x.FullName == "Package.Package.Package.Hello"));
+            Assert.IsTrue(entries.Any(x => x.FullName == "SignedPackage.SignedPackage.SignedPackage.Hello"));
         }
 
+        //FYI SingedPackage zero-touch code
+        //namespace SignedPackage
+        //{
+        //    public class SignedPackage
+        //    {
+        //        public string Hello()
+        //        {
+        //            return nameof(Hello);
+        //        }
+        //    }
+        //}
```

Comment:
Can you put this comment above the Assert above instead of outside of this test?